controllers/keystoneapi_controller.go

```diff
@@ -605,7 +606,7 @@ func (r *KeystoneAPIReconciler) reconcileConfigMap(ctx context.Context, instance
 
 	secretName := "openstack-config-secret"
 	var openStackConfigSecret keystone.OpenStackConfigSecret
-	openStackConfigSecret.Clouds.Default.Auth.Password = string(keystoneSecret.Data["AdminPassword"])
+	openStackConfigSecret.Clouds.Default.Auth.Password = instance.Spec.PasswordSelectors.Admin
```

Comment:
Should this be `string(keystoneSecret.Data[instance.Spec.PasswordSelectors.Admin])` since `instance.Spec.PasswordSelectors.Admin` holds the selector (the key) for the password and not the value of the password itself?

Reply:
yes it should